src/features/editor/index.tsx

```diff
@@ -17,58 +17,84 @@ import { CorpusContainer, NamedContainers, Verse } from 'structs';
 import '../../styles/theme.css';
 import BCVWP from '../bcvwp/BCVWPSupport';
 import { AlignmentSide } from '../../common/data/project/corpus';
-import { SuggestionsContext, useSuggestionsContextInitializer } from '../../hooks/useSuggestions';
+import {
+  SuggestionsContext,
+  useSuggestionsContextInitializer,
+} from '../../hooks/useSuggestions';
 import { useAppSelector } from '../../app/index';
 
+export type CustomEditorStyles = Partial<{
+  polyglot: Record<string, unknown>;
+  controlPanel: Record<string, unknown>;
+  contextPanel: Record<string, unknown>;
```

Comment:
`Record<string, unknown>` should be replaced with `Partial<SxProps<Theme>>`, except in the case of `contextPanel` which should be `Partial<React.CSSProperties>`.